src/GafferSceneUI/SelectionTool.cpp

```diff
@@ -197,12 +197,12 @@ bool SelectionTool::buttonPress( const GafferUI::ButtonEvent &event )
 	PathMatcher selection = sg->getSelection();
 
 	const bool shiftHeld = event.modifiers && ButtonEvent::Shift;
+	const bool controlHeld = event.modifiers && ButtonEvent::Control;
```

Comment:
Should be a bitwise & (the line above should be too, but that's fixed on master I think).

Reply:
Arg, of course. Blindly copying is never a good idea. Thanks!